docs/source/validation_framework.rst

```diff
@@ -6,7 +6,7 @@ a pretty good job validating the syntax but it does not provide any tooling to p
 be described in XSD 1.0 (PyXB does not understand XSD 1.1 and xpath assertions).
 
 Example:
-    if tt:tt has timeBase=clock smpteTimingType must not be present anywhere in the document.
+    If tt:timebase="clock" then all begin and end attributes of the subtitle shall be of type ebuttdt:clockTimingType.
```

Comment:
needs to be `ttp:timebase`; would also change the wording of "the subtitle" to "timed elements".